pkg/kubectl/main.go

```diff
@@ -17,6 +18,13 @@ import (
 
 func Main() {
 	kubenv := os.Getenv("KUBECONFIG")
+	for i, arg := range os.Args {
+		if strings.Contains(arg, "--kubeconfig=") {
+			kubenv = strings.Split(arg, "=")[1]
+		} else if strings.Contains(arg, "--kubeconfig") {
+			kubenv = os.Args[i+1]
```

Comment:
Should we perform any additional check here in case the next value is empty or outside the range of the slice?

Reply:
Added